src/FcoeClient.ycp

```diff
@@ -782,7 +792,7 @@ VLAN interface to get a valid configuration."), fcoe_vlan_interface) );
 		info_map = add( info_map, "device", card["model"]:"");		// model
 	    }
 		
-	    info_map = add( info_map, "fcoe_vlan", fcoe_vlan_interface );	// FCoE VLAN interface, e.g. eth3.200
+	    info_map = add( info_map, "fcoe_vlan", humanized_interface(fcoe_vlan_interface) );	// FCoE VLAN interface, e.g. eth3.200
```

Comment:
Um, to avoid another possible translation issues I'd suggest translate the value at the presenter level, i.e. when creating the dialog. Putting translated values right into the internal structure is fragile.

If you change the language during installation (Back, Back,...) then the displayed values might not reflect the language change if the detected devices are cached somewhere.

I do not know how much difficult would be to refactor this but if it is easy I'd prefer to move the translations to the user interaction level.